subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/FileContentGenerator.groovy

```diff
@@ -109,9 +109,14 @@ abstract class FileContentGenerator {
             }
 
             return includedProjects + """
-            if(org.gradle.util.GradleVersion.current() >= org.gradle.util.GradleVersion.version("4.6")) {
-            ${config.featurePreviews.collect { "enableFeaturePreview(\"$it\")" }.join("\n")}
-            }
+                def enableFeaturePreviewSafe(String feature) {
```

Comment:
I guess this only works for Groovy settings and we never generate Kotlin `settings.gradle.kts`, right?

Reply:
Right.